src/services/business-lookup-services.ts

```diff
@@ -36,7 +36,7 @@ export default class BusinessLookupServices {
     const legalType = 'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,SP,CC,XL,FI,XCP,PA'
     const url = this.businessApiUrl +
       `businesses/search/facets?start=0&rows=20&categories=legalType:${legalType}::status:ACTIVE` +
-      `&query=${encodeURIComponent(query)}`
+      `&query=value:${encodeURIComponent(query)}`
```

Comment:
If you haven't already, please update the other ticket for this change to indicate that you've already updated this UI. (You could also update Edit UI and complete the other ticket ;) )

Reply:
I have assigned that ticket (#13145) to my name. Updating edit-ui.